libs/pinecone/tests/unit_tests/test_embeddings.py

```diff
@@ -83,7 +116,7 @@ def test_custom_config(self) -> None:
         """Test custom configuration overrides defaults."""
         embeddings = PineconeEmbeddings(
             model=MODEL_NAME,
-            api_key=API_KEY,
+            pinecone_api_key=API_KEY,
```

Comment:
same again, breaking change - let's add a copy of some tests and in the copy we use the alternative method (ie one test with `api_key` and one test with `pinecone_api_key` - let's do this for all instances (index, rerank, embed)